MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -897,7 +897,7 @@ private void BuildStreamVideoItem(
 
             var appliedVideoConditions = options.Profile.CodecProfiles
                 .Where(i => i.Type == CodecType.Video &&
-                    i.ContainsAnyCodec(videoStream?.Codec, container) &&
+                    i.ContainsAnyCodec(videoCodec, container) &&
```

Comment:
Reviewing #9374, now I think we probably need to use `playlistItem.VideoCodecs` in place of `videoCodec` because the latter remains unchanged - without HLS restrictions (https://github.com/nyanmisaka/jellyfin/blob/7bd971ffefe983a1f32836b0f2fcbe602f92efa0/MediaBrowser.Model/Dlna/StreamBuilder.cs#L806-L809).

Reply:
That makes sense. I'll start a new one.

Reply:
Also, here https://github.com/jellyfin/jellyfin/blob/2fe13f54eaf87eefefd27f4ccb2ace1371f5e886/MediaBrowser.Model/Dlna/StreamBuilder.cs#L907

Or maybe just assign it back?
```cs
videoCodec = playlistItem.VideoCodecs;
```